File: android/app/src/main/java/com/ode/appe/SplashActivity.java
package com.ode.appe;

import android.app.Activity;

import android.content.Intent;
import android.os.Bundle;

public class SplashActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, MainActivity.class);
        intent.putExtras(this.getIntent());
        startActivity(intent);
        finish();
    }
}

File: android/app/src/main/java/com/ode/appe/SplashActivity.kt
package com.ode.appe

import android.app.Activity
import android.content.Intent
import android.os.Bundle

class SplashActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent(this, MainActivity::class.java)
        intent.putExtras(this.intent)
        startActivity(intent)
        finish()
    }
}
